.github/workflows/ci_linting.yml

```diff
@@ -144,7 +144,7 @@ jobs:
         with:
           route: GET /repos/{repo}/commits/{ref}/statuses?per_page=100
           repo: ${{ github.repository }}
-          ref: ${{ github.event.merge_group.head_sha || github.event.pull_request.head.sha }}
+          ref: ${{ github.event.merge_group.head_sha || github.event.pull_request.head.sha || github.event.after }}
```

Comment:
It seems like the syntax should be `github.event.push.after`, but I tested this on my fork and `ref` was still undefined. You can see `after` defined directly in `event` when [printing the contents of `github.event`](https://github.com/goatgoose/s2n-tls/actions/runs/13142410076/job/36672487123#step:3:6)

Reply:
It makes me very sad that this is different for like every event...